CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Changed
+
+- Remove decorations (dots at the beginning of each line) from output to stdout in admin commands.
+  Now you need to add a line break (`\n`) yourself when using the `print()` function in your commands.
```

Comment:
I think this is unnecessary. I suggest using `Println()` to print data to stdout.

Reply:
done